infrastructure/ops/prod/k8s_repo/config/kustomization-ops.yaml

```diff
@@ -14,6 +14,7 @@ bases:
 
 resources:
   - istio-operator-psp.yaml
+  - istio-telemetry.yaml
```

Comment:
Remember if it is here there are 2 considerations:
1. You have to have build_repo.sh copy it where you need it to be
2. It will be deployed from the get go.

Reply:
updated